wallets/provider-walletconnect-2/src/session.ts

```diff
@@ -153,9 +152,9 @@ export async function tryConnect(
   const pairing = tryGetPairing(client);
   if (pairing) {
     try {
-      session = await restoreSession(client, pairing);
+      session = await restoreSession(client, pairing.topic);
     } catch (e) {
-      await disconnectSessions(client);
+      await disconnectSessions(client).catch((error) => console.log(error));
```

Comment:
This can be problematic. why you are ignoring the error? it wouldn't reject `tryConnect` promise if fails, which means it continues to the next lines. Is there any reason?

Reply:
Yes, you are right. Adding this catch in disconnect was enough.